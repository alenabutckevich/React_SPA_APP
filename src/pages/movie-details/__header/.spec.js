import React from 'react';
import MovieDetailsHeader from './';

describe('<MovieDetailsHeader />', () => {

    it('render movie details header', () => {
        const wrapper = shallow(<MovieDetailsHeader/>);
       
        expect(wrapper).toMatchSnapshot();
    })
})