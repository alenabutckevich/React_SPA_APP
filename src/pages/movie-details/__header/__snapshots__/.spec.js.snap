// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MovieDetailsHeader /> render movie details header 1`] = `
<div>
  <a
    className="search-link"
    onClick={[Function]}
  >
    search
  </a>
  <MovieAnnotation />
</div>
`;
